controllers/host/platformnetworks.go

```diff
@@ -1110,6 +1130,79 @@ func (r *HostReconciler) ReconcilePlatformNetworkPrincipalSimplex(
 
 }
 
+func (r *HostReconciler) ReconcilePlatformNetworkPrincipalMultinodesOAM(
+	client *gophercloud.ServiceClient,
+	host_instance *starlingxv1.Host,
+	host *v1info.HostInfo,
+	network_instance *starlingxv1.PlatformNetwork,
+	addrpool_instance *starlingxv1.AddressPool,
+	system_info *cloudManager.SystemInfo) error {
+
+	system_addrpool, err := GetSystemAddrPool(client, addrpool_instance)
+	if err != nil {
+		return err
+	}
+
+	_, update_required, _ := r.IsAddrPoolUpdateRequired(network_instance, addrpool_instance, system_addrpool)
+
+	validation_result := r.ValidateAddressPool(network_instance, addrpool_instance, system_info)
+
+	if update_required && validation_result {
+
+		active_controller_instance, active_controller_obj, err := r.CloudManager.GetHostByPersonality(host_instance.Namespace, client, cloudManager.ActiveController)
```

Comment:
We have to set both active and standby controller's StrategyRequired to "lock_required" so that VIM takes that into account during strategy build phase.
During OAM reconfiguration lock and unlock is required on both the controllers.

Reply:
That's the reason I don't like to differ the "active controller" and "standby controller" if the operation is expected only on "controllers" rather than other personalities, can you add comments on that?

Reply:
Done